resources/linux/code-url-handler.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Name=@@NAME_LONG@@ - URL Handler
+Comment=Code Editing. Redefined.
+GenericName=Text Editor
+Exec=/usr/share/@@NAME@@/@@NAME@@ --open-url %U
+Icon=@@ICON@@
+Type=Application
+NoDisplay=true
+StartupNotify=true
+StartupWMClass=@@NAME_SHORT@@
```

Comment:
This makes the icon to appear 2 times on Gnome Shell side panel. If I have it as pinned as favorite and click on it, it appears as if it was another application. If this line is commented off the error disappears.

![image](https://user-images.githubusercontent.com/3135638/47703111-6050db80-dbdc-11e8-98a9-0c93d78bb071.png)